Automattic/pocket-casts-ios
# Referrals: Claim pass

| 📘 Part of: #2083  |  

This PR implements only the UI for the claim passes flow. The signup and IAP flows will be done on a separate PR
At the moment because we don't have any server control logic the banner to claim passes will always be present .

| 1 | 2 |
| - | - |
| ![Simulator Screenshot - iPhone 15 Pro - 2024-09-11 at 00 03 52](https://github.com/user-attachments/assets/ae434310-0eba-4129-b42a-cc19ba38da9f) | ![Simulator Screenshot - iPhone 15 Pro - 2024-09-11 at 00 04 01](https://github.com/user-attachments/assets/6c41ccd0-12c2-4004-b891-385f9662294f) | 
| ![Simulator Screenshot - iPad mini (6th generation) - 2024-09-11 at 00 09 21](https://github.com/user-attachments/assets/5dd8f4f2-bbb4-4065-b4e3-141a96fac7b8) | ![Simulator Screenshot - iPad mini (6th generation) - 2024-09-11 at 00 09 25](https://github.com/user-attachments/assets/dfb1318a-724b-41d8-af22-25dc2b6c72b4) |

## To test

1. Start the app
2. Ensure that you have the referrals FF enabled
3. Open profile
4. Check that you see the banner for claim passes
5. Tap on it
6. See that you see the Claim pass view in a sheet
7. Tap on not now
8. Check that you are returned to Profile

## Checklist
